### PR TITLE
Feature/server email verification

### PR DESCRIPTION
**Pull request by nom0786**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,10 +15,12 @@
     "express-validator": "^5.3.1",
     "i": "^0.3.6",
     "mysql": "^2.16.0",
+    "nodemailer": "^6.0.0",
     "npm": "^6.4.1",
     "passport": "^0.4.0",
     "passport-local": "^1.0.0",
-    "prop-types": "^15.6.2"
+    "prop-types": "^15.6.2",
```
Comment (mackosx) [inline package.json]: i dont think you added this but we dont need prop-types in the server package.json

File: server/src/routes/api/users.js [hunk 2]
```diff
@@ -85,20 +87,28 @@ router.post('/signup', async (req, res) => {
             console.log(err);
           }
 
+          //create the token here and store in DB
+           const token = mail.generateEmailToken();
```
Comment (mackosx) [inline server/src/routes/api/users.js]: 🥇

File: package.json (hunk 1 above)
Comment (mackosx) [reply package.json]: Also you should rerun npm install in the client to get the package-lock.json file back and commit it

**Comment by mackosx** [comment]
Also merge latest dev and resolve conflicts

File: package.json (hunk 1 above)
Comment (nom0786) [reply package.json]: kk done

**Comment by nom0786** [comment]
most of the stuff should be updated for merging

**Comment by mackosx** [comment]
looks like just linting errors left

**Pull request merged**